### ISSUE TITLE
Retrospection [For Week5 Lab]

### ISSUE THREAD

**Issue by alphonsekoh** [ICT2x01-P3-4/ict2x01-p3-4#35]
## Retrospection
For this week lab and the labs for the past few weeks, I have learnt the importance of planning when embarking on a project. It gives the team a common goal and through the labs for use case diagrams and class diagrams to component and sequence diagrams it aids us to identify functions and components, allowing the us to split the workload evenly and have more in-depth view of what we need to do in our software functions. 

In conclusion, the lab lessons allows me to understand more about the software architecture in the SDLC and allowing my team to plan more efficiently.
